src/main/java/screens/login_screen/UserLoginUI.java

```diff
@@ -1,20 +1,37 @@
 package screens.login_screen;
+import data_access.Database;
+import data_access.UserDatabase;
+import interface_adapters.login_interface_adapters.UserChatsPresenter;
+import interface_adapters.login_interface_adapters.UserLoginViewI;
 import use_cases.user_login_use_cases.UserLoginInputBoundary;
-import use_cases.user_registration_use_cases.UserVerificationOutputBoundary;
+import interface_adapters.login_interface_adapters.UserLoginPresenter;
+import use_cases.user_login_use_cases.UserLoginInteractor2;
+import interface_adapters.user_registration_interface_adapters.UserVerificationOutputView;
 
 import javax.swing.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.io.File;
 
 /** This is the screen on which the user enters his credentials in order to login **/
-public class UserLoginUI implements ActionListener, UserVerificationOutputBoundary {
+public class UserLoginUI implements ActionListener, UserVerificationOutputView {
 
-    private final UserLoginInputBoundary loginInteractor;
+    private final UserLoginPresenter loginPresenter;
     JTextField credentialText;
     JPasswordField passwordText;
 
-    public UserLoginUI(UserLoginInputBoundary loginInteractor){
-        this.loginInteractor = loginInteractor;
+    public UserLoginUI(UserLoginPresenter loginPresenter){
+        UserLoginViewI loginViewI = new AppScreenCreator();
+        this.loginPresenter = loginPresenter;
+        this.loginPresenter.setLoginView(loginViewI);
+    }
+    //For Testing Purposes
+    public static void main(String[] args){
+        Database userDB = new UserDatabase(new File("new"));
```

Comment:
Instead of new File("new") could you maybe have it as new FIle("user_accounts") instead? I think everywhere else in our code we have it referenced as "user_accounts" so we would need to change it in all those places

Reply:
Sure